### ISSUE TITLE
autodecline when finding match.

### ISSUE THREAD

**Issue by jas8669** [ValveSoftware/Dota-2#2615]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): 
* i7 10 series.  6650xt gpu.  32 gigs ram.  garuda linux sway.
* Have you checked for system updates?: [Yes/No]
* yes
* Are you using the latest stable video driver available for your system? [Yes/No]
* yes
* Have you verified the game files?: [Yes/No]
* no

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large pastes as a [Github Gist](https://gist.github.com/).

every time i look for a game, the second it finds one it auto declines.  been like this for months.  since the TI update.
#### Steps for reproducing this issue:

1. 
2. 
3.

**Comment by rbnagm** [comment]
I experienced a similar behavior one evening on a specific patch. But it didn't autodecline. The game coordinator didn't respond to my client specifically until the time frame to accept timed out. I know that because one of my friends in party queue didn't experience the issue and saw some of us not accepting. Affected machines had different OS, some Windows, some Linux.

In our case we just restarted the game multiple times, until the issue didn't persist. It took us 1.5 hours to do this, but we had no issues since then.

Can you create a Custom game on the public game servers?

**Comment by jas8669** [comment]
> I experienced a similar behavior one evening on a specific patch. But it didn't autodecline. The game coordinator didn't respond to my client specifically until the time frame to accept timed out. I know that because one of my friends in party queue didn't experience the issue and saw some of us not accepting. Affected machines had different OS, some Windows, some Linux.
> 
> In our case we just restarted the game multiple times, until the issue didn't persist. It took us 1.5 hours to do this, but we had no issues since then.
> 
> Can you create a Custom game on the public game servers?

yes i can host a custom game on public( i think never done it before, but it appears to be the case.)

also i have had the issue you mention in your post, but that can be resolved fairly easily like you mention.  this has been going on for months now without any resolution.

**Comment by steakscience** [comment]
Also getting this, basically the Accept Match popup never appears

**Comment by jas8669** [comment]
> Also getting this, basically the Accept Match popup never appears

are you getting this still after the christmas/frostivus update?  also distor/DE are you using?

i still am.  this is has been going on for like 4 months at this point.  it's ridiculous since valve is trying to push linux.

**Comment by steakscience** [comment]
@jas8669 Nope

**Comment by jas8669** [comment]
> @jas8669 Nope

that's good.  i thought mine went away but is back again.  please reply again if it comes back.  what distro/de are you using?  i might think about switching if it's different.  i'm on an arch based system.

**Comment by steakscience** [comment]
> > @jas8669 Nope
> 
> that's good. i thought mine went away but is back again. please reply again if it comes back. what distro/de are you using? i might think about switching if it's different. i'm on an arch based system.

Arch btw, Hyprland

It's a network issue, I doubt it's system related. When I had this issue, it turns out the nearest Steam coordinator server was messed up.

**Comment by jas8669** [comment]
> Replying to https://github.com/ValveSoftware/Dota-2/issues/2615#issuecomment-1858766473

nice.

i use an arch based system too, so that must not be it.

i don't think it's a server issue on their end, i have a windows partition i use just for dota now and it always works.  i can go back and forth and it works on windows and only works 1/10 or 1/20 times on linux.  i can even load windows, get into a game, then restart and play on linux.  but can never accept a game with linux.
